presto-main/src/test/java/io/prestosql/memory/TestLocalMemoryManager.java

```diff
@@ -17,63 +17,66 @@
         assertFalse(localMemoryManager.getReservedPool().isPresent());
         assertEquals(localMemoryManager.getPools().size(), 1);
     }
 
     @Test
-    public void testReservedMemoryEnabled()
+    public void testReservedMemoryPoolEnabled()
     {
         NodeMemoryConfig config = new NodeMemoryConfig()
                 .setHeapHeadroom(new DataSize(10, GIGABYTE))
                 .setMaxQueryMemoryPerNode(new DataSize(20, GIGABYTE))
                 .setMaxQueryTotalMemoryPerNode(new DataSize(20, GIGABYTE));
 
-        LocalMemoryManager localMemoryManager = new LocalMemoryManager(config,
-                new DataSize(60, GIGABYTE).toBytes());
+        LocalMemoryManager localMemoryManager = new LocalMemoryManager(config, new DataSize(60, GIGABYTE).toBytes());
         assertTrue(localMemoryManager.getReservedPool().isPresent());
         assertEquals(localMemoryManager.getPools().size(), 2);
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class)
-    public void testMaxQueryMemoryPerNodeBiggerQueryTotalMemoryPerNode()
+    @Test
+    public void testMaxQueryMemoryPerNodeBiggerThanQueryTotalMemoryPerNode()
     {
         NodeMemoryConfig config = new NodeMemoryConfig()
                 .setHeapHeadroom(new DataSize(10, GIGABYTE))
                 .setMaxQueryMemoryPerNode(new DataSize(200, GIGABYTE))
                 .setMaxQueryTotalMemoryPerNode(new DataSize(20, GIGABYTE));
 
-        new LocalMemoryManager(config, new DataSize(60, GIGABYTE).toBytes());
+        assertThatThrownBy(() -> new LocalMemoryManager(config, new DataSize(60, GIGABYTE).toBytes()))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageMatching("Max query memory per node \\(query.max-memory-per-node\\)" +
+                        " cannot be greater than the max query total memory per node \\(query.max-total-memory-per-node\\).*");
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class,
-            expectedExceptionsMessageRegExp = "Invalid memory configuration.* " +
-                    "The sum of max total query memory per node .* and heap headroom .*" +
-                    "cannot be larger than the available heap memory .*")
+    @Test
     public void testNotEnoughAvailableMemory()
     {
         NodeMemoryConfig config = new NodeMemoryConfig()
                 .setHeapHeadroom(new DataSize(10, GIGABYTE))
                 .setMaxQueryMemoryPerNode(new DataSize(20, GIGABYTE))
                 .setMaxQueryTotalMemoryPerNode(new DataSize(20, GIGABYTE));
 
-        new LocalMemoryManager(config, new DataSize(10, GIGABYTE).toBytes());
+        assertThatThrownBy(() -> new LocalMemoryManager(config, new DataSize(10, GIGABYTE).toBytes()))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageMatching("Invalid memory configuration.* " +
```

Comment:
Looks like the message is `Invalid memory configuration. The sum of` so this can be
```java
"Invalid memory configuration\\. The sum of"